pkg/providers/snow/apibuilder.go

```diff
@@ -45,6 +45,12 @@ func KubeadmControlPlane(clusterSpec *cluster.Spec, snowMachineTemplate *snowv1.
 		fmt.Sprintf("/etc/eks/bootstrap.sh %s %s", clusterSpec.VersionsBundle.Snow.KubeVip.VersionedImage(), clusterSpec.Cluster.Spec.ControlPlaneConfiguration.Endpoint.Host),
 	)
 
+	if snowMachineTemplate.Spec.Template.Spec.ContainersVolume != nil {
+		kcp.Spec.KubeadmConfigSpec.PreKubeadmCommands = append(kcp.Spec.KubeadmConfigSpec.PreKubeadmCommands,
```

Comment:
I was under the impression that we only wanted to support this for worker nodes and not for CP

Reply:
i confirmed w snow team, cp volume is also supported